### PR TITLE
Tasks 1315-136 and 1188-1194

### PR DESCRIPTION
**Pull request by Ravsehajsinghpuri**

### CODE REVIEW

**Comment by aarunku5** [comment]
Thanks, change your task numbers from 1188-1194.
I'll list corrections here according to you original task numbers though:

1168-
Definition- split the first sentence into 2 sentences

1169-
Split the definition into 3 sentences.

1170-
Split the definition into 2 sentences.
In your input, put a comma after the list so format it as- { [....], k=..}

1171-
Capitalize "Indian"
Split the definition into 2 sentences
Hyphenate "non-vegetarian"

1172-
Capitalize "Indian"
Split the definition into 2 sentences

1173-
Capitalize "Indian"
Split the definition into 2 sentences
classify the dish whether-->classify the dish as a

1174-
Split definition into 2 sentences.

In tasks 1171-1173, add one sentence to specify the output format, i.e., "Your output must be in the form of the words __/__" (based on task output).

**Comment by Ravsehajsinghpuri** [comment]
@aarunku5 thanks for the review, working on these changes now. Regarding the last comment, In tasks 1171-1173, add one sentence to specify the output format, i.e., "Your output must be in the form of the words /" (based on task output).
How should I mention the words non vegetarian or vegetarian. Should it be something like /non vegetarian/ /vegetarian/. Because I cannot write a string inside a string. What would you suggest on this?

**Comment by aarunku5** [comment]
You can use the backslash to put them in single quotes as a list- \'non-vegetarian\' (or) \'vegetarian\'

**Comment by Ravsehajsinghpuri** [comment]
I have made almost all the changes. @aarunku5 Please review and merge

**Comment by aarunku5** [comment]
Thanks, just a couple more edits

1190-
Split the definition into 2 sentences.
In the examples and instances, can you format the input like this:
[100, 291, 285, 281, 151, 189, 220, 235, 288, 160, 233, 198, 294, 269, 162, 152, 217, 247, 98, 145]** , ** k=14

**Comment by Ravsehajsinghpuri** [comment]
Sure, will make these edits. Thanks!

**Comment by Ravsehajsinghpuri** [comment]
@aarunku5 is this good to merge now?

**Comment by Palipoor** [comment]
Hi! Can you please remove the files 1146 - 1152? As the task numbers are now 1188-1194.

**Comment by Ravsehajsinghpuri** [comment]
@Palipoor not sure what you mean by this. These tasks have already been merged into master.

**Review by Palipoor — APPROVED** [review]
LGTM now!

**Comment by Ravsehajsinghpuri** [comment]
perfect! Thank a lot @Palipoor for doing this. I know things were inconsistent somewhere, glad you fixed them.

File: tasks/task1148_maximum_ascii_value.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
         {
             "input": "ABXyOMnde", 
             "output": "X",
-            "explanation": "y is the character with the maximum ascii value and not X."
+            "explanation": "X is not the character with maximum ascii value in the given string."
```
Comment (danyaljj) [inline tasks/task1148_maximum_ascii_value.json:29]: thanks!

**Pull request merged**